napalm/base.py

```diff
@@ -364,3 +364,9 @@ def get_interfaces_counters(self):
             }
         """
         raise NotImplementedError
+
+        def cli(self, command = ''):
```

Comment:
Could we set that to None and specify in the docstring that it has to be a list of commands?